packages/maskbook/src/plugins/ITO/hooks/useClaimCallback.ts

```diff
@@ -73,7 +81,7 @@ export function useClaimCallback(pid: string) {
                 reject(error)
             })
         })
-    }, [account, chainId, ITO_Contract])
+    }, [account, chainId, ITO_Contract, pids])
```

Comment:
Better sort and stringify `pids` in case of redundant rerenders.

Reply:
I think any changes of pids leads to rerenders is preferable here.